graphene_django/rest_framework/tests/test_mutation.py

```diff
@@ -204,20 +205,23 @@ class Meta:
             serializer_class = MySerializer
 
     # missing required fields
-    result = MyMutation.mutate_and_get_payload(None, mock_info(), **{})
-    assert len(result.errors) > 0
+    with raises(ValidationError):
+        result = MyMutation.mutate_and_get_payload(None, mock_info(), **{})
+        assert len(result.errors) > 0
```

Comment:
Does this assert ever get called? If I understand correctly line 209 should raise a `ValidationError` thus line 210 never gets called.

Reply:
No it doesn't. I think line 210 is unnecessary indeed.